SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -165,7 +163,7 @@ void SpatialVirtualWorkerTranslator::ConstructVirtualWorkerMappingFromQueryRespo
 // to the spatialOS storage.
 void SpatialVirtualWorkerTranslator::SendVirtualWorkerMappingUpdate()
 {
-	if (!bIsReady || NetDriver == nullptr)
+	if (NetDriver == nullptr)
```

Comment:
Is this ever okay to just return safely? This logic happens after having had multiple interactions with SpatialOS, should we `check(NetDriver)` instead?

Reply:
As below, this is for testing.